shift_allocation/tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .nh_clinical_staff_allocation import *
+from .nh_clinical_staff_reallocation import *
+from .nh_clinical_doctor_allocation import *
+from . import test_operations
+
+# Disabled Tests
+# from . import test_user_allocation
+# from . import test_allocation_wizards
+# from . import test_allocation_responsibility_allocation
+# from . import test_allocation_unfollow
+# from . import test_get_allocation_locations
+# from . import test_responsibility_allocation_wizard
+# from . import test_user_responsibility_allocation
```

Comment:
Ah okay that's because `nh_clinical/tests/test_auditing.py` was an existing disabled test that contained an allocation related test. Even though it was disabled I still moved the test to the new module and duplicated the setup.

I could spend time cutting down the setup so that only what is necessary is in there but if the test is broken anyway there would be no easy way to tell if I've cut down the setup correctly without actually fixing the test (because how do I know a failure is because of faulty setup vs broken test?). I've added a comment to this new, semi-duplicated test to explain what happened and that there is probably unnecessary stuff in the setup. Does that sound okay?

Reply:
Created Task EOBS-2433 to run all disabled tests and identify why they are disabled. Happy leaving it as it is for this PR as it has been confirmed that the original file from where this test came is currently disabled. :shipit: